ui/components/errors/unexpected-error.js

```diff
@@ -1,13 +1,14 @@
 import HTTPStatus from 'http-status';
 import PropTypes from 'prop-types';
 import React from 'react';
+import { Link } from '../../routes';
 
 export default function UnexpectedError({ statusCode }) {
   return (
     <section>
       <h1>{HTTPStatus[statusCode] || 'Unexpected error'}</h1>
       <p>
-        Visit our <a href="/">homepage</a> or <a href="mailto:ofcio@omb.eop.gov">contact us</a> if
+        Visit our <Link route="homepage"><a>homepage</a></Link> or <a href="mailto:ofcio@omb.eop.gov">contact us</a> if
```

Comment:
Question to ponder: this error message will appear if something unexpected went wrong (an API call failed, some exception was thrown, etc.), but the app is still running (so Next/React are working okay). Do you think it'd be safer to avoid the AJAXy links here?

Reply:
@cmc333333 I agree. Ill make an update.